kratos/modeler/connectivity_preserve_modeler.h

```diff
@@ -125,6 +131,18 @@ class KRATOS_API(KRATOS_CORE) ConnectivityPreserveModeler : public Modeler
         const Condition& rReferenceCondition
     );
 
+
+    void SetupModelPart() override;
```

Comment:
I don't want to be annoying, but can you add doxygen?

Reply:
Added, but I seem to recall that Doxygen was smart enough to get the docstrings from the base class if available.